backend/routers/social_media_marketing_router.py

```
@@ -104,55 +95,42 @@ class SupabaseStorageService: pass
     class ThemeService: pass
-    # CORE.MD: Define a fallback dependency to prevent NameError if the import fails.
     def get_theme_service():
         raise HTTPException(status_code=501, detail="Theme service is not available.")
     
-# CORE.MD: Import the new StabilityAiService for dynamic avatar generation.
 try:
     from services.stability_ai_service import StabilityAiService, get_stability_service
     STABILITY_SERVICE_AVAILABLE = True
 except ImportError:
     STABILITY_SERVICE_AVAILABLE = False
     class StabilityAiService: pass
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)